package.json

```diff
@@ -45,11 +47,12 @@
     "fs-extra": "^8.1.0",
     "jest": "^26.4.2",
     "jolocom-lib": "^5.2.1",
+    "log4js": "^6.3.0",
```

Comment:
have you compared this against other options for logging libraries? Why did you choose log4js in specific?

Reply:
yes, I compered with others (with [winston](https://www.npmjs.com/package/winston) specifically) and [log4js](https://www.npmjs.com/package/log4js) more lightweight (~x3 times), has standardized log structure out of the box, easier configurable and faster (for example [those](https://www.loggly.com/blog/a-benchmark-of-five-node-js-logging-libraries/) results)